crates/core/tedge_mapper/src/az/mapper.rs

```diff
@@ -30,6 +33,38 @@ impl TEdgeComponent for AzureMapper {
     ) -> Result<(), anyhow::Error> {
         let (mut runtime, mut mqtt_actor) =
             start_basic_actors(self.session_name(), &tedge_config).await?;
+        if tedge_config.mqtt.bridge.built_in {
+            let remote_clientid = tedge_config.device.id.try_read(&tedge_config)?;
+            let rules = built_in_bridge_rules(remote_clientid)?;
+
+            let mut cloud_config = rumqttc::MqttOptions::new(
+                remote_clientid,
+                tedge_config.az.url.or_config_not_set()?.to_string(),
+                8883,
```

Comment:
Something is wrong here. I would expect call to `host()` and `port()` on `tedge_config.az.url`.

Reply:
`tedge_config.az.url` is a `ConnectUrl`, which explicitly doesn't support a port. We know the port is 8883 because that is what Azure uses for MQTT connections.